src/hotspot/share/gc/g1/g1ConcurrentRefine.cpp

```diff
@@ -251,13 +251,19 @@ static size_t calc_min_yellow_zone_size() {
 }
 
 static size_t calc_init_green_zone() {
-  size_t green = G1ConcRefinementGreenZone;
-  const char* name = "G1ConcRefinementGreenZone";
+  size_t green;
   if (FLAG_IS_DEFAULT(G1ConcRefinementGreenZone)) {
-    green = ParallelGCThreads;
-    name = "ParallelGCThreads";
+    // rate_1 is a relatively conservative guess at the rate for pause-time
+    // card refinement by one thread.
+    const double rate_1 = 200.0; // cards/ms/thread
```

Comment:
I would prefer if this magic number were placed somewhere more standing out than in some static method in a cpp file. I thought at least put it into the .hpp file as part of the `G1ConcurrentRefine" class. However there are no guidelines for that for g1 code where or how to collect these magic constants I'm kind of good with that too.

Reply:
I agree with Thomas. I think this should live in `G1ConcurrentRefine`, but if you don't agree I'm ok with leaving as is.

Reply:
Moving it into the class would have required moving the referencing static
function into the class, &etc. I've given it a more informative name and
description, and moved it to file scope to make it more prominent.